apps/web/components/Forms/EventsForm.tsx

```diff
@@ -31,10 +31,29 @@ export const EventsForm = ({ handleSubmit }: EventsFormProps) => {
         console.log('Errors here', errors);
         return (
           <Form>
-            <TextInput name="title" handleChange={handleChange} value={values.title} error={errors.title} />
-            <TextInput name="summary" handleChange={handleChange} value={values.blurb} error={errors.blurb} />
-            <TextInput name="location" handleChange={handleChange} value={values.address} error={errors.address} />
-            <TextInput
+            <Field
+              component={TextInput}
+              name="title"
+              handleChange={handleChange}
+              value={values.title}
+              error={errors.title}
```

Comment:
you dont need these props (i dont think)

Reply:
```jsx
<Field component={TextInput} name="title" />
```

Reply:
Just pushed this change!